mix.exs

```diff
@@ -26,7 +26,8 @@ defmodule Exnumerator.Mixfile do
 
   defp deps do
     [
-      {:ex_doc, "~> 0.16", only: :dev, runtime: false},
+      {:ecto, ">= 0.0.0", only: [:dev, :test]},
```

Comment:
Why do you need `ecto` here?

Reply:
Probably in the future yes.

Reply:
https://github.com/KamilLelonek/exnumerator/blob/210e7f94bf9ac487b3d008fcd4df75cd8afad5a4/lib/exnumerable.ex#L4

Reply:
I would remove that line if you do not want Ecto as deps.

Reply:
What solution is better in your opinion?

Reply:
@KamilLelonek  Add `ecto` as optional, it will be downloaded for testing but you will have to download your own ecto when you use it as deps.